src/amo/components/Link.js

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+
+
+export class InternalLinkBase extends React.Component {
```

Comment:
nit: could this just be `LinkBase`? That's the convention we've [mostly] been following since the module is `Link.js`.